modules/migration-known-issues.adoc

```diff
@@ -17,18 +17,10 @@ This release has the following known issues:
 +
 These annotations preserve the UID range, ensuring that the containers retain their file system permissions on the target cluster. There is a risk that the migrated UIDs could duplicate UIDs within an existing or future namespace on the target cluster. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1748440[*BZ#1748440*])
 
-* If an AWS bucket is added to the {mtc-short} web console and then deleted, its status remains `True` because the `MigStorage` CR is not updated. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1738564[*BZ#1738564*])
-
 * Most cluster-scoped resources are not yet handled by {mtc-short}. If your applications require cluster-scoped resources, you might have to create them manually on the target cluster.
 
 * If a migration fails, the migration plan does not retain custom PV settings for quiesced pods. You must manually roll back the migration, delete the migration plan, and create a new migration plan with your PV settings. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1784899[*BZ#1784899*])
 
 * If a large migration fails because Restic times out, you can increase the `restic_timeout` parameter value (default: `1h`) in the `MigrationController` CR.
 
 * If you select the data verification option for PVs that are migrated with the file system copy method, performance is significantly slower.
-
-ifeval::["{mtc-version}" < "1.4"]
-* If you are migrating data from NFS storage and `root_squash` is enabled, `Restic` maps to `nfsnobody`. The migration fails and a permission error is displayed in the `Restic` pod log. You can resolve this issue by creating a supplemental group for Restic. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1873641[*BZ#1873641*])
```

Comment:
Please, can double confirm that this issue regarding the NFS must be removed in this PR?

Reply:
The NFS bit above seems to be unrelated to the BZ linked to this PR. It's possible that some other change has made the NFS note no longer needed, but I am unaware of any change of that nature. I suspect this should still remain.

Reply:
@sseago  and @sergiordlr 

This text was not removed as part of the bug. This text was removed because it is for versions earlier than MTC 1.4.

The conditional text is a remnant from the days before I had merge rights for OCP. Since OCP and MTC did not have the same release schedule, I had to mark text with conditions to ensure that it did not appear in versions where the bug has been fixed. Therefore you can ignore these two removed bullet items.

Reply:
@apinnick The BZ fix still requires setting supplemental groups for restic, though. The BZ fix referenced just makes that easier to do. Even with MTC 1.4, the supplemental groups must be set for restic, but as of 1.4, this can be done via operator settings. As long as this need is documented elsewhere, it can probably be removed here, but if this is the only reference to it, then we still need this (but perhaps without the "<1.4" part).

Reply:
@sseago  How is this done in the Operator settings? It would be good to add this info somewhere else. If it is only mentioned in Known Issues, most people will not find it until it is too late.

Reply:
@apinnick Yes, Step 8 there describes what's needed. You can see Dylan's comment on the merged PR for more details on what the user would need to do: https://github.com/konveyor/mig-operator/pull/442 Basically, the migrationcontroller CR needs to have the following spec field added:
```
  spec:
  ...
    restic_supplemental_groups:
    - 5555
    - 6666
```
If the above two supplemental groups need to be added to the restic pods.

Reply:
@sseago 
I created a new bug to track the Restic workaround: https://bugzilla.redhat.com/show_bug.cgi?id=1964305
Thanks.